source/views/tasks_view.py

```diff
@@ -4,22 +4,36 @@
 from schemas import *
 # from sqlalchemy_pagination import paginate
 
+from utils import *
+
+from utils import *
+
+
 
 class TasksView(FlaskView):
     task_schema = TaskSchema()
 
+    class TasksPaginationSchema(PaginationSchema):
+        items = fields.List(fields.Nested(TaskSchema))
+
+    pagination_schema = TasksPaginationSchema()
+
     def index(self):
-        tasks = Task.query.all()
-        # page = Task.query.order_by(Task.paginate(1,10,False))
-        tasks_data = self.task_schema.dump(tasks, many=True).data
-        return jsonify({'task': tasks_data}), 200
+        user = authorization(request.headers.get('Authorization', None))
+        page = request.args.get('Page', None)
+        per_page = request.args.get('Per_page', None)
+        tasks = Task.query.paginate(page, per_page, error_out=False)
+        tasks_data = self.pagination_schema.dump(tasks).data
+        return jsonify(tasks_data), 200
 
     def get(self, id_task):
+        user = authorization(request.headers.get('Authorization', None))
```

Comment:
Only assign the return value of authorizartion when need user info in the method, otherwise you can only call the authorization method